java/src/jmri/managers/configurexml/AbstractTurnoutManagerConfigXML.java

```diff
@@ -187,23 +187,23 @@ public boolean loadTurnouts(Element shared, Element perNode) {
         try {
             if (shared.getChild("defaultclosedspeed") != null) {
                 String closedSpeed = shared.getChild("defaultclosedspeed").getText();
-                if (closedSpeed != null && !closedSpeed.equals("")) {
+                if (closedSpeed != null && !closedSpeed.isEmpty()) {
                     tm.setDefaultClosedSpeed(closedSpeed);
                 }
             }
         } catch (jmri.JmriException ex) {
-            log.error(ex.toString());
+            log.error("TuroutManager {} {}", tm, ex.getMessage());
```

Comment:
I don't think the managers have a `toString()` method that gives anything useful except the class name. And the class name is shown anyway in the log. So I suggest removing the manager from the message.

TurnoutManager is misspelled. Turout.